.changeset/clever-buckets-exist.md

```diff
@@ -0,0 +1,11 @@
+---
+'@clerk/backend': patch
+---
+
+Fix the following `@clerk/backend` methods to populated paginated responses:
```

Comment:
Not sure what this means. Can we improve the wording here?

Reply:
fixed! PTAL